controllers/v1/template.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * name : wishlist.js
+ * author : Saish Borkar
+ * created-date : 28-Nov-2024
+ * Description :  template Controller.
+ */
+
+// dependencies
+const libraryCategoriesHelper = require(MODULES_BASE_PATH + "/library/categories/helper");
+
+/**
+ * UserExtension service.
+ * @class
+ */
+
+module.exports = class Template  {
+
+	static get name() {
+		return 'template'
+	}
+
+	/**
+* @api {post} /project/v1/template/list
+* @apiVersion 1.0.0
+* @apiName add
+* @apiGroup template
+* @apiHeader {String} X-user-token Authenticity token
+* @apiUse successBody
+* @apiUse errorBody
+* @apiParamExample {json} Response:
+* {
+    "message": "Successfully fetched projects",
+    "status": 200
+}
+*/
+	async list(req) {
```

Comment:
@borkarsaish65 add validation for  function if required. and also add api configuration and raise utils PR as well

Reply:
done